backend/main.py

```
@@ -732,97 +732,84 @@ async def analyze_ticket(request_body: TicketRequest, request: Request):
-        "ocr_text": request_body.image_text or "",
-        "image_description": ""
-    }
-    
-    if request_body.image_base64 and not gemini_analysis["ocr_text"]:
+    gemini_analysis = {"ocr_text": request_body.image_text or "", "image_description": ""}
+    if request_body.image_base64 and not gemini_analysis["ocr_text"] and gemini_service:
         try:
-            print("[AI] Detecting visual context via Gemini...")
             vision_result = gemini_service.analyze_image(request_body.image_base64, text)
             gemini_analysis.update(vision_result)
         except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)